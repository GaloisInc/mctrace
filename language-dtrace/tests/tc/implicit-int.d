proc:::exec_success {
  global_int = 0;
}
