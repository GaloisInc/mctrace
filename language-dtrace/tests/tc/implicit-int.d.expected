TCExpected { globals = [("global_int", BVType 32)]
         }
